apps/web/playwright/fixtures/regularBookings.ts

```diff
@@ -357,5 +361,84 @@ export function createBookingPageFixture(page: Page) {
       await scheduleSuccessfullyPage.waitFor({ state: "visible" });
       await expect(scheduleSuccessfullyPage).toBeVisible();
     },
+    checkAvailabilityTab: async () => {
+      const editAvailability = (await localize("en"))("edit_availability");
+
+      // verify if the icon is rendered
+      await expect(page.locator("span").filter({ hasText: "Europe/London" }).locator("svg")).toBeVisible();
+      await expect(page.getByText("Europe/London")).toBeVisible();
+      await page.getByRole("link", { name: editAvailability }).click();
+    },
+    goToAvailabilityPage: async () => {
+      const workingHours = (await localize("en"))("default_schedule_name");
+
+      await page.goto("/availability");
+      await page
+        .getByTestId("schedules")
+        .locator("div")
+        .filter({
+          hasText: workingHours,
+        })
+        .first()
+        .click();
+    },
+    checkAvailabilityPage: async () => {
+      const sunday = (await localize("en"))("sunday");
+      const monday = (await localize("en"))("monday");
+      const wednesday = (await localize("en"))("wednesday");
+      const saturday = (await localize("en"))("saturday");
+      const save = (await localize("en"))("save");
+
+      await page.getByTestId("availablity-title").click();
+      // change availability name
+      await page.getByTestId("availablity-title").fill("Working Hours test");
+      await expect(page.getByText("Mon - Fri, 9:00 AM - 5:00 PM")).toBeVisible();
+      await page.locator("label").filter({ hasText: sunday }).getByRole("switch").click();
+      await page.locator("div").filter({ hasText: monday }).first().click();
+      await page.locator("label").filter({ hasText: wednesday }).getByRole("switch").click();
+      await page.locator("label").filter({ hasText: saturday }).getByRole("switch").click();
+      await page
+        .locator("div")
+        .filter({ hasText: "Sunday9:00am - 5:00pm" })
+        .getByTestId("add-time-availability")
+        .first()
+        .click();
+      await expect(page.locator("div").filter({ hasText: "6:00pm" }).nth(1)).toBeVisible();
+      await page.getByRole("button", { name: save }).click();
+      await expect(page.getByText("Sun - Tue, Thu - Sat, 9:00 AM - 5:00 PM")).toBeVisible();
+      await expect(page.getByText("Sun, 5:00 PM - 6:00 PM")).toBeVisible();
+      await page
+        .locator("div")
+        .filter({ hasText: "Sunday9:00am - 5:00pm" })
+        .getByTestId("copy-button")
+        .first()
+        .click();
+      await expect(
+        page.getByText("Copy times toSelect AllSundayMondayTuesdayWednesdayThursdayFridaySaturday")
```

Comment:
it would be great if we could try to avoid using anything targeting text directly, can we replace this using getByTestId ?

Reply:
We will try that